scripts/setup-k8s-master.sh.in

```diff
@@ -78,6 +78,7 @@ bootstrapTokens:
   - authentication
 controllerManagerExtraArgs:
   cloud-provider: aws
+  configure-cloud-routes: "false"
```

Comment:
Do we know why this seems to be required for v1.11.x and not for v1.10.x? As far as I can tell the default for this hasn't changed.

Reply:
I tried yesterday with 1.10.x and controller manager also shows the same error message

Reply:
Okay, so digging a bit further, I wonder if we should be changing the value that we set for `allocate-node-cidrs` instead of changing `configure-cloud-routes`.